server/src/dtos/shared-link.dto.ts

```diff
@@ -30,6 +30,10 @@ export class SharedLinkCreateDto {
   @Optional()
   password?: string;
 
+  @IsString()
+  @Optional()
+  shareSlug?: string;
```

Comment:
I think I'd probably vote for `slug`. That matches the database column and the share part is also kind of implicit in a shared link dto.

Reply:
Easy enough.